python/example_robot_data/robots_loader.py

```diff
@@ -58,6 +58,7 @@ class RobotLoader(object):
     has_rotor_parameters = False
     free_flyer = False
     verbose = False
+    model_path = None
```

Comment:
Perhaps, it is a good idea to start adding some documentation for these hyper-parameters.